azure-pipelines.yml

```diff
@@ -21,6 +21,13 @@ steps:
     projects: '$(solution)'
     configuration: '$(buildConfiguration)'
 
+
+- task: CmdLine@2
+  displayName: 'Run Azure Storage Emulator'
+  inputs:
+    script: '"%ProgramFiles(x86)%\Microsoft SDKs\Azure\Storage Emulator\AzureStorageEmulator.exe" start'
```

Comment:
Did this end up working?

Reply:
The .yml file is valid. But seems like the test cases are failing in CI/CD. I am looking into it